tests/core/actions/test_forms.py

```diff
@@ -1863,12 +1863,7 @@ async def test_extract_slots_with_mapping_conditions_during_form_activation():
     )
     assert events == []
 
-    expected_events = [
-        ActiveLoop(form_name),
-        SlotSet(slot_name, entity_value),
```

Comment:
This test shouldn't be amended because the slot gets filled by a different action (the default `action_extract_slots`) and not by the form action as in your test change.